src/main/kotlin/org/rust/cargo/runconfig/CargoRunConfigurationBase.kt

```diff
@@ -46,14 +56,11 @@ class CargoCommandConfiguration(project: Project,
         }
     }
 
-    override fun getConfigurationEditor(): SettingsEditor<out RunConfiguration> =
-        CargoRunConfigurationEditorForm()
-
     override fun getState(executor: Executor, environment: ExecutionEnvironment): RunProfileState? {
         val module = configurationModule.module ?: return null
         val toolchain = module.toolchain ?: return null
         val moduleDirectory = module.cargoProjectRoot ?: return null
-        val args = ParametersListUtil.parse(additionalArguments)
+        val args = ParametersListUtil.parse(arguments + additionalArguments)
```

Comment:
> sorry for letting that slip.

No need to say sorry, it is impossible to write correct code and everybody makes bugs :)